src/main/resources/db/migration/V1_60__YORKSHIRE_AND_THE_HUMBER_ONBOARDING.sql

```diff
@@ -0,0 +1,175 @@
+INSERT INTO team (code, name, ldu_code) VALUES('N55DS3', 'Doncaster Sentence Management Team 3', 'N55DONL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55DS4', 'Doncaster Sentence Management Team 4', 'N55DONL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55DS5', 'Doncaster Sentence Management Team 5', 'N55DONL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55AA4', 'Awaiting Allocation Doncaster', 'N55NDR');
+
+INSERT INTO team (code, name, ldu_code) VALUES('N55KEA', 'Awaiting Allocation East Yorkshire', 'N55NHEY');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HCA', 'Awaiting Allocation Hull', 'N55NHEY');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS7', 'Beverley Sentence Management Team 7', 'N55EYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS8', 'Bridlington Sentence Management Team 8', 'N55EYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55ECI', 'East Riding Community Integration Team', 'N55EYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS9', 'Goole Sentence Management Team 9', 'N55EYL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HC1', 'Hull Community Integration Team 1', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HC2', 'Hull Community Integration Team 2', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HC3', 'Hull Community Integration Team 3', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HC4', 'Hull Community Integration Team 4', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS1', 'Hull Sentence Management Team 1', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS2', 'Hull Sentence Management Team 2', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS3', 'Hull Sentence Management Team 3', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS4', 'Hull Sentence Management Team 4', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS5', 'Hull Sentence Management Team 5', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55HS6', 'Hull Sentence Management Team 6', 'N55HUL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55N55', 'Hull Sentence Management Team 10', 'N55HUL');
+
+INSERT INTO team (code, name, ldu_code) VALUES('N55AAK', 'Awaiting Allocation Kirklees', 'N55NKS');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS5', 'Kirklees Community Integration Team', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS1', 'Kirklees Sentence Management Team 1', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS2', 'Kirklees Sentence Management Team 2', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS3', 'Kirklees Sentence Management Team 3', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KS4', 'Kirklees Sentence Management Team 4', 'N55KIRL');
+INSERT INTO team (code, name, ldu_code) VALUES('N55KUW', 'UPW- Kirklees Sentence Management Team 5', 'N55KIRL');
```

Comment:
Does this want an extra space before the hyphen?

Reply:
good spot!